products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -251,11 +251,13 @@ public struct CommandMetrics {
 public struct CommandExtendedResult {
     public let result: CommandResult    /// The result of a command execution
     public let exitStatus: Int32        /// The exit code
+    public let pid: Int32               /// The process identifier (-1 if failed)
```

Comment:
Given that we are bridging to swift here, I'd say make this an optional that is nil when the process ID doesn't exist.  Also, though not 100% certain ("I found it on the internet"), I think we can also use pid_t here in swift.

Reply:
pid_t is an typealias for Int32, but you're right - it's better to be explicit here! :)